config/runtime/fstests.jinja2

```diff
@@ -36,6 +36,16 @@ class Job(BaseJob):
         except Exception as e:
             print('Kernel build error:', e)
 
+    def _run_smoke_tests(self):
+        try:
+            result = subprocess.run(["kvm-xfstests", "smoke"], check=True)
```

Comment:
As a follow-up, the other `subprocess.run()` calls should be cleaned up in the same way.